CRUDRecipeEF.PL/Program.cs

```diff
@@ -24,43 +26,22 @@ private static void Main(string[] args)
                 var context = services.GetRequiredService<RecipeContext>();
                 if (!context.Recipes.Any() && !context.Ingredients.Any())
                 {
-                    DataSeed.Seed(context);
+                    var seeder = host.Services.GetRequiredService<IDataSeed>();
+                    seeder.Seed();
                 }
             }
             catch (Exception ex)
             {
-                Console.WriteLine("Retarded error while seeding data");
+                contextLog.Fatal("Unable to seed database: {exception}", ex.Message);
```

Comment:
shouldnt it be ex and not exception or where is this variable from?

Reply:
It's not a variable it's a serilog thing